libs/ui/primitives/src/textarea.tsx

```diff
@@ -2,9 +2,9 @@ import * as React from 'react'
 
 import { cn } from '@nmit-coursition/ui/utils'
 
-export interface TextareaProps extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {}
+export interface RootProps extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {}
 
-const Textarea = React.forwardRef<HTMLTextAreaElement, TextareaProps>(({ className, ...props }, ref) => {
+const Main = React.forwardRef<HTMLTextAreaElement, RootProps>(({ className, ...props }, ref) => {
```

Comment:
**suggestion:** We should never touch `primitives` unless Shadcn updates it. It can cause conflicts.